### PR TITLE
[demo] Slowloading of images

### PR DESCRIPTION
**Pull request by klucsik**
flagd.json and frontend/frontend proxy env var changes to include flagd
increased frontend pod ram limit
connected PRs:
https://github.com/open-telemetry/opentelemetry-demo/pull/1515
https://github.com/open-telemetry/opentelemetry-demo/pull/1571

### CODE REVIEW

File: charts/opentelemetry-demo/values.yaml [hunk 1]
```diff
@@ -381,6 +385,10 @@ components:
         value: "4318"
       - name: OTEL_COLLECTOR_HOST
         value: $(OTEL_COLLECTOR_NAME)
+      - name: FLAGD_HOST
```
Comment (puckpuck) [inline charts/opentelemetry-demo/values.yaml:388]: I don't believe the frontend-proxy (envoy) needs the FLAGD environment variables.

Comment (klucsik) [reply charts/opentelemetry-demo/values.yaml:388]: It cannot expose flagd to the browser clients (frontend client side flag provider) without those.

**Review by puckpuck — APPROVED** [review]
This is good to merge!

**Comment by jaronoff97** [comment]
@klucsik can you rebase when you get a chance?

**Comment by klucsik** [comment]
> @klucsik can you rebase when you get a chance?

Rebased, and squashed commits

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**